po/vi.po

```diff
@@ -6254,138 +6254,168 @@ msgid ""
 "[`Vec`](https://doc.rust-lang.org/std/vec/struct.Vec.html) is the standard "
 "resizable heap-allocated buffer:"
 msgstr ""
+"[`Vec`](https://doc.rust-lang.org/std/vec/struct.Vec.html) là kiểu dữ liệu "
+"chuẩn dạng mảng có thể tự thay đổi kích thước:"
 
 #: src/std-types/vec.md
 msgid "\"v1: len = {}, capacity = {}\""
-msgstr ""
+msgstr "\"v1: độ dài = {}, kích thước = {}\""
 
 #: src/std-types/vec.md
 msgid "\"v2: len = {}, capacity = {}\""
-msgstr ""
+msgstr "\"v2: độ dài = {}, kích thước = {}\""
 
 #: src/std-types/vec.md
 msgid "// Canonical macro to initialize a vector with elements.\n"
-msgstr ""
+msgstr "// Macro để khởi tạo một vector với nhiều phần tử cho trước.\n"
 
 #: src/std-types/vec.md
 msgid "// Retain only the even elements.\n"
-msgstr ""
+msgstr "// Chỉ giữ lại các phần tử chẵn.\n"
 
 #: src/std-types/vec.md
 msgid "\"{v3:?}\""
-msgstr ""
+msgstr "\"{v3:?}\""
 
 #: src/std-types/vec.md
 msgid "// Remove consecutive duplicates.\n"
-msgstr ""
+msgstr "// Loại bỏ các phần tử liên tiếp giống nhau.\n"
 
 #: src/std-types/vec.md
 msgid ""
 "`Vec` implements [`Deref<Target = [T]>`](https://doc.rust-lang.org/std/vec/"
 "struct.Vec.html#deref-methods-%5BT%5D), which means that you can call slice "
 "methods on a `Vec`."
 msgstr ""
+"`Vec` implements [`Deref<Target = [T]>`](https://doc.rust-lang.org/std/vec/"
```

Comment:
Translate "implements" please!

Reply:
`Deref<Target = [T]>` here is a trait, and `Vec` is a struct, in this context `Vec implements Deref<Target = [T]>` actually means `Vec` has a method called `deref`. You can read more about the `Deref` trait [here](https://doc.rust-lang.org/std/ops/trait.Deref.html).
Therefore, this `implement` is not an actual English verb, it's a Rust term